bindings/python/src/pre_tokenizers.rs

```diff
@@ -489,17 +489,71 @@ impl PyMetaspace {
         setter!(self_, Metaspace, add_prefix_space, add_prefix_space);
     }
 
+    #[getter]
+    fn get_prepend_scheme(self_: PyRef<Self>) -> String {
+        // Assuming Metaspace has a method to get the prepend_scheme as a string
+        let scheme: PrependScheme = getter!(self_, Metaspace, get_prepend_scheme());
+        match scheme {
+            PrependScheme::First => "first",
+            PrependScheme::Never => "never",
+            PrependScheme::Always => "always",
+        }
+        .to_string()
+    }
+
+    #[setter]
+    fn set_prepend_scheme(self_: PyRef<Self>, prepend_scheme: &str) -> PyResult<()> {
+        // Assuming Metaspace has a method to set the prepend_scheme from a string
+        let scheme = match prepend_scheme.to_lowercase().as_str() {
```

Comment:
```suggestion
        let scheme = match prepend_scheme.as_str() {
```

I know we like to be very accepting of parameters, but refusing things makes code overall for everyone more consistent which is better in my eyes (unless backward compat forces something like this)